examples/c/ex_all.c

```diff
@@ -866,6 +866,10 @@ transaction_ops(WT_CONNECTION *conn, WT_SESSION *session)
 	/*! [set oldest timestamp] */
 	error_check(conn->set_timestamp(conn, "oldest_timestamp=2a"));
 	/*! [set oldest timestamp] */
+
+	/*! [rollback to stable] */
+	error_check(conn->rollback_to_stable(conn, "timestamp=2a"));
+	/*! [rollback to stable] */
```

Comment:
I see the problem, I've opened WT-3492 for followup work.  Suggest we just fix this call here by changing the config string to `NULL`.